server/graphql/v1/mutations/orders.js

```diff
@@ -393,7 +393,7 @@ export async function createOrder(order, loaders, remoteUser, reqIp) {
 
     // If using a tier, amount can never be less than the minimum amount
     if (tier && tier.presets) {
-      const minValue = min(isNil(tier.amount) ? tier.presets : [...tier.presets, tier.amount]);
+      const minValue = tier.minimumAmount || min(isNil(tier.amount) ? tier.presets : [...tier.presets, tier.amount]);
```

Comment:
Can we stick to just use `tier.minimumAmount` here and assume it's set?

Reply:
Yes we can